hermeznetwork/hermez-node
# Feature/#765 info msg

Closes #765

### What does this PR does?

This PR adds error codes and type to the info field in the tx table.

### How to test?

Run the hermez node and send an invalid tx to see the message. Also, you can run the tests to check it.

Batch forged: https://explorer.internaltestnet.hermez.io/batch/25226

## Checklist

These are the criteria that every PR should meet, please check them off as you
review them:
